howto.tex

```diff
@@ -296,3 +296,14 @@ \subsection{Authentication}\label{sec:auth}
 Two generic set of credentials to access the Jira REST API and the Jira database have been defined.
 These credentials are available at \texttt{1password.com}, in the LSST-IT architecture vault, but not yet integrated into docsteady.
 In order to use these credentials, they have to be configured using environment variables, added as options from the command line, or entered when prompted, as specified in this technical note.
+
+In addition, Jenkins DSL scripts use the \texttt{rubinobs-dm-admin} credential-ID defined in the \textit{Manage Credentials} Jenkins section.
+
+
+\subsection{Deployment of New Versions}\label{sec:deploy}
+
+The \texttt{docs-ci} Jenkins application is hosted on NCSA Kubernetes cluster. The deployment files are available at
+\url{https://github.com/docs-ci/docs-ci-kubektl}.
```

Comment:
Should be `kubectl`, no?  (You can rename repositories, but I don't know if there are other things depending on the name.)